ccpp/driver/GFS_diagnostics.F90

```diff
@@ -1144,15 +1144,17 @@ subroutine GFS_externaldiag_populate (ExtDiag, Model, Statein, Stateout, Sfcprop
 
     idx = idx + 1
     ExtDiag(idx)%axes = 2
-    ExtDiag(idx)%name = 'snowc_ave'
-    ExtDiag(idx)%desc = 'snow cover - GFS lsm'
+    ExtDiag(idx)%name = 'snowc'
+    !ExtDiag(idx)%name = 'snowc_ave'
+    ExtDiag(idx)%desc = 'snow cover '
     ExtDiag(idx)%unit = '%'
     ExtDiag(idx)%mod_name = 'gfs_phys'
-    ExtDiag(idx)%time_avg = .TRUE.
+    !ExtDiag(idx)%time_avg = .TRUE.
```

Comment:
The ExtDiag(idx)%time_avg and ExtDiag(idx)%cnvfac are used to get averaged snow cover (snowc_ave). the Instantaneous snowc is a different field, please do not replace the field as there is impact on downstream users. You can add a new variable "snowci" for instantaneous snow cover.

Reply:
Ok, a new variable is added for instantaneous snow cover fraction.

Reply:
Yes, Jun's comment is correct.  Changing it will have major impact on downstream models, forecasters and community users.

Reply:
@yangfanglin @junwang-noaa Original time-averaged sow cover fraction is restored.